tests/cypress/tests/create_policy_field_validation.spec.js

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2020 Red Hat, Inc. */
+/// <reference types="cypress" />
+
+describe('RHACM4K-2349 - GRC UI: [P1][Sev1][policy-grc] Create policy page: Check policy field validations', () => {
+
+  const ID = Cypress.env('RESOURCE_ID')
+  const errorMsgRegExp = /Invalid name due to Kubernetes naming restriction.The name must meet the following requirements:• contain no more than 253 characters• contain only lowercase alphanumeric characters, '-' or '.'• start with an alphanumeric character• end with an alphanumeric character/
+//  const errorMsgRegExp = /Invalid name due to Kubernetes naming restriction.The name must meet the following requirements:/
+  const alertMsgRegExp = /Name already exists: pressing 'Create' will prompt to update the existing policy/
+  const namePatterns = [
+                         'this-is-n,ot-a-valid-name',
+                         '-this-is-not-a-valid-name',
+                         'this-is-not-a-valid-name-',
+                         'this-is-a-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-long-name-that-is-too-long-and-should-not-work-when-its-put-into-the-name-field'
+                       ]
+  // while we could use the next definition, due to a bug bz#1936431 better use a unique policy name
+  //const longValidName = 'this-is-a---really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-long-valid-name-that-should-work-when-its-put-into-the-name-field'`
+  const longValidName = `this-is-a---really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-really-long-valid-name-that-should-work-when-its-put-into-the-name-field-${ID}`
+
+  it('Check that invalid policy name pattern issues an error', () => {
```

Comment:
@kkaarreell This would be duplicate test of https://github.com/open-cluster-management/grc-ui/blob/main/tests/cypress/tests/InvalidYaml_governance.spec.js#L11, You can extend the existing test and use. I'll leave this to you.

Reply:
It is not a duplicate. Your test validates content coming from YAML while this test validates content entered in the form. So it is close but not really identical. I think we should test both and can keep these separated.